app/crypto/asymmetric.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Asymmetric
```

Comment:
```suggestion
class Crypto::RSA
```
maybe RSA is better in that case. it's shorter.
only Asymmetric is not an option since it does not include information about encryption. 
e.g. there's asymmetric downloads